### PR TITLE
Create PR_17

### PR DESCRIPTION
**Pull request by TestGitHub100500**
PR_17

### CODE REVIEW

**Comment by OksanaEfanova** [comment]
comment

File: PR_17 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (OksanaEfanova) [inline PR_17:1]: commet1

**Review by OksanaEfanova — CHANGES_REQUESTED** [review]

File: PR_17 (hunk 1 above)
Comment (TestGitHub100500) [reply PR_17:1]: commet2

Comment (OksanaEfanova) [reply PR_17:1]: commet2

**Pull request closed without merging**